src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
```

Comment:
instead of setting the height for the header set padding/height for items - it will help you with active line positioning